tests/unit/src/main/scala/tests/TestingServer.scala

```diff
@@ -1339,7 +1339,16 @@ final class TestingServer(
           PrettyPrintTree(
             label(uri),
             parent.nodes
-              .map(n => PrettyPrintTree(label(n.nodeUri)))
+              .map { n =>
```

Comment:
One of the tests for TreeViewLSP suite is failing on Windows. Added something to debug the issue, hopefully it helps.